src/components/input/DebouncedTextField.tsx

```diff
@@ -0,0 +1,86 @@
+import { useEffect, useState } from 'react';
+
+import { TextField } from '@mui/material';
+
+import { useBuilderTranslation } from '@/config/i18n';
+import { hooks } from '@/config/queryClient';
+import { DEBOUNCED_TEXT_FIELD_ID } from '@/config/selectors';
+import { BUILDER } from '@/langs/constants';
+
+const { useDebounce } = hooks;
+
+export const DEBOUNCE_MS = 1000;
+
+type Props = {
+  initialValue?: string;
+  placeholder?: string;
+  label: string;
+  mt?: number;
+  canBeEmpty?: boolean;
+  emptyValueError?: string;
+  onUpdate: (newValue?: string) => void;
+};
+
+export const DebouncedTextField = ({
+  initialValue,
+  placeholder,
+  label,
+  mt,
+  canBeEmpty = true,
```

Comment:
```suggestion
  required = true,
```
I think it matches the usual props?

Reply:
Yeah you right ! :)